src/main/java/org/apache/commons/lang3/time/StopWatch.java

```diff
@@ -671,4 +677,68 @@ public void unsplit() {
         splitState = SplitState.UNSPLIT;
     }
 
+    <T, E extends Throwable> T get(FailableSupplier<T, E> supplier) throws E {
```

Comment:
Add Javadoc with a `@since` tag.

Reply:
Added, but not sure if the version is correct. Please let me know if I have to adjust it.